src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixLegacyResourceDesignerStep.cs

```diff
@@ -144,6 +148,8 @@ protected override void FixBody (MethodBody body, TypeDefinition designer)
 						instructions.Add (i, newIn);
 					} else {
 						LogMessage ($"DEBUG! Failed to find {key}!");
+						var msg = string.Format (CultureInfo.CurrentCulture, Resources.XA_Fixup_Designer_Error, key.Replace ("::", "/"));
```

Comment:
Would be handy to have a comment with a "sample key" containing `::` so that we have an idea of what this will look like.